src/helpers/helpers.js

```diff
@@ -1,2 +1,17 @@
 export const getTrendingVideos = videos =>
   [...videos].sort((video1, video2) => video2.views - video1.views);
+
+export const getUniqueValues = (data, type) => {
+  let unique = data.map(item => item[type]);
+  return ["All", ...new Set(unique)];
+};
+
+export const getFilteredVideos = (videos, state) => {
+  if (state.filters.category !== "All") {
+    return [...videos].filter(
```

Comment:
I think the filter does not mutate the original Array so no need to destructure again

Reply:
yes. I did this for extra caution.